### PR TITLE
Add <optgroup> example

### PR DESCRIPTION
**Pull request by wbamberg**
Please let me know if you think this is too much! It does have a slightly serious point, though, that in user testing some people thought examples like this were part of the page.

### CODE REVIEW

**Comment by wbamberg** [comment]
Added an image from Unsplash with acceditation, as in https://github.com/mdn/interactive-examples/pull/712.

File: live-examples/html-examples/css/optgroup.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.output {
+    background: top/cover no-repeat url("/media/examples/dino2.jpg");
```
Comment (schalkneethling) [inline live-examples/html-examples/css/optgroup.css]: Nit: Code Style: single quotes

File: live-examples/html-examples/css/optgroup.css [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    position: relative;
+}
+
+footer {
+    font-size: .8rem;
+    position: absolute;
+    bottom: 10px; left: 10px;
```
Comment (schalkneethling) [inline live-examples/html-examples/css/optgroup.css]: Nit: Code Style: Each selector on its own line

File: live-examples/html-examples/css/optgroup.css (hunk 1 above)
Comment (schalkneethling) [inline live-examples/html-examples/css/optgroup.css]: File name should be `dino.jpg`

**Pull request merged**